src/components/InstallerView.jsx

```diff
@@ -0,0 +1,31 @@
+import viewStyles from '../styles/view'
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/helpers/i18n'
+import classNames from 'classnames'
+
+class InstallerView extends Component {
+  doInstall () {
+    // get the values from the fields
+    const slug = document.querySelector('#slug').value
+    const repourl = document.querySelector('#repourl').value
+    const isupdate = document.querySelector('#isupdate').checked
+    this.onSubmit(slug, repourl, isupdate)
+  }
+
+  render ({t, onSubmit}) {
```

Comment:
Passing `props` in the `render`'s parameter is not react compatible.
By the way, a props is accessible in the whole component instance.
So there is no need to `this.onSubmit = onSubmit`.